src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -222,6 +261,10 @@ public void setMatriculationNumber(MatriculationNumber matriculationNumber) {
             return Optional.ofNullable(matriculationNumber);
         }
 
+        /**
```

Comment:
I believe that the javadoc for the param covidStatus is missing.

Reply:
Thank you for pointing out the error! I have fixed it 👍